feature-libs/product-configurator/rulebased/components/group-menu/configurator-group-menu.component.ts

```diff
@@ -86,13 +85,19 @@ export class ConfiguratorGroupMenuComponent {
   ) {}
 
   /**
-   * Fired on key board events, checks for 'enter' and delegates to click.
+   * Fired on key board events, checks for 'enter' or 'space' and delegates to click.
    *
    * @param {KeyboardEvent} event - Keyboard event
    * @param {Configurator.Group} group - Entered group
    */
+  preventScrollingOnSpace(event: KeyboardEvent): void {
```

Comment:
Please, write a comment for this method.

Reply:
Done